docs/tutorials/modify_configuration.md

```diff
@@ -1,15 +1,89 @@
-# Modify Configuration Parameter
-
-- a. Explain how configuration parameters are simulation wide parameters.
-- b. Explain how settings in other areas (campaign and demographics) can impact configuration parameters and vice versa.
-- c. Explain how you can sub-class frames so that changes in the base class impact the sub-classes
-- d. Use extend_frame to create a new frame off of baseline called higher_base_infectivity
-- e. Explain who each file of the frame has two parts, but we will focus on just the initialize_xxx() function for now.
-- f. Set Base_Infectifivity
-- g. Run EMOD
-- h. Use the plotting commands to plot InsetChart and compare to baseline
-- i. Tell the user to demonstrate the power of the base class/frame, we will change the Simulation_Duration in baseline
-- j. Run EMOD with baseline
-- k. Plot InsetChart of new baseline against previous baseline and previous higher_base_infectivity
-- l. Run EMOD with higher_base_infectivity
-- m. Plot InsetChart of new_higher_base_infectivity against previous higher_base_infectivitycd
+# Modify Configuration Parameters
+
+## What are Configuration Parameters?
+
+Configuration parameters are simulation-wide parameters that affect how EMOD runs at a fundamental level. These include
+parameters related, but not limited to:
+
+* model duration, timestep length, and when the model starts
+* how a disease progresses and transmits
+* whether agents can be born, age, and/or die
+* output report specifications
+
+## Modifying a Configuration Parameter
+
+Configuration parameters are assigned values either in a country model or in a project frame, in one of two functions
+(in both locations): `initialize_config` and `get_config_parameterized_calls`.
+
+### Example
+
+In this example we update the value of configuration parameter named **Base_Infectivity**, which alters how infectious
+a disease is at a fundamental level (before any other modifiers apply).
+
+#### Create a new project and baseline frame
+
+```bash 
+python -m emodpy_workflow.scripts.new_project -d configuration_tutorial
+cd configuration_tutorial
+python -m emodpy_workflow.scripts.new_frame --country Zambia --dest baseline
+```
+
+#### Extend the baseline frame for alteration
+
+```bash 
+python -m emodpy_workflow.scripts.extend_frame --source baseline --dest more_infectious
+```
+
+#### Edit the frame config.py
+
+Here we edit the `initialize_config` function, opting to not make a hyperparameter for setting `Base_Infectivity`.
```

Comment:
Is it worth describing why you've opted not to make it a hyperparameter here or is that too big of a topic? Maybe just link to the topic that describes hyperparameters more?